addons/account/wizard/account_move_send_batch_wizard.py

```diff
@@ -31,11 +33,20 @@ def default_get(self, fields_list):
     @api.depends('move_ids')
     def _compute_summary_data(self):
         sending_methods = dict(self.env['res.partner']._fields['invoice_sending_method'].selection)
+        extra_edis = self._get_all_extra_edis()
+
         for wizard in self:
-            wizard.summary_data = {
-                sending_method: {'count': len(moves), 'label': sending_methods[sending_method]}
-                for sending_method, moves in wizard.move_ids.grouped(self._get_default_sending_method).items()
-            }
+            summary_data = dict()
+            edi_counter = Counter()
+
+            for sending_method, moves in wizard.move_ids.grouped(self._get_default_sending_method).items():
+                summary_data[sending_method] = {'count': len(moves), 'label': sending_methods[sending_method]}
+                edi_counter += Counter([edi for move in moves for edi in self._get_default_extra_edis(move)])
```

Comment:
[suggestion] Not a big fan of this line. Can we rewrite in a sexier way ? If not, maybe a little comment ?